README.md

```diff
@@ -1,3 +1,25 @@
+---
+page_type: sample
+languages:
+- azdeveloper
+- csharp
+- powershell
+- shell
+- html
+- bicep
+products:
```

Comment:
Add products: Redis Cache, App Configuration, Key Vault.  If the samples generally cover it, the pattern also has hub-spoke, networking, private link